backends/platform/sdl/sdl.cpp

```diff
@@ -800,5 +801,8 @@ char *OSystem_SDL::convertEncoding(const char *to, const char *from, const char
 	memcpy(finalResult, result, newLength + zeroBytes);
 	SDL_free(result);
 	return finalResult;
+#else
+	return nullptr;
```

Comment:
This should call the base implementation `ModularBackend::convertEncoding` (it only returns `nullptr` now but could always change in the future)

Reply:
Done